pkg/storage/SQL/operational_solution/and_possible_get_by_operational_need_id_LOADER.sql

```diff
@@ -29,7 +29,8 @@ possibleSolution AS (
         COALESCE(OpNd.created_by, '00000000-0000-0000-0000-000000000000') AS created_by, -- This is UUID.NIL, the same as the UNKNOWN_USER account in the DB
         COALESCE(OpSol.created_dts, CURRENT_TIMESTAMP) AS created_dts,
         OpSol.modified_by,
-        OpSol.modified_dts
+        OpSol.modified_dts,
+        TRUE AS is_common_solution
```

Comment:
Can you add comments to these? I think it might just help clarify exactly how this is determined in each case. Maybe for here, something like

```suggestion
        TRUE AS is_common_solution -- Possible Solutions are always a common solution
```

Reply:
Good call, thanks Clay.